frontend/src/pages/AccountSend.vue

```diff
@@ -568,6 +570,19 @@ function useSendForm() {
         gasLimit: sendMax.value && sendingNativeToken ? sendMaxGasLimit : undefined,
       });
       void txNotify(tx.hash, ethersProvider);
+      if (viewingKeyPair.value?.privateKeyHex && userAddress.value) {
+        await storeSend({
+          recipientAddress: recipientId.value,
+          chainId: chainId.value!,
+          advancedMode: advancedMode.value,
+          viewingKey: viewingKeyPair.value?.privateKeyHex,
+          amount: tokenAmount.toString(),
+          tokenAddress,
+          hash: tx.hash,
+          userAddress: userAddress.value,
+          checkbox: advancedAcknowledged.value,
+        });
+      }
```

Comment:
We will have the sent data even if the transaction fails. We could move this after tx.wait(), but would lose data if a user leaves or refreshes the page.